src/components/JobConfiguration.vue

```diff
@@ -7,6 +7,12 @@
 
     <ion-list>
 
+      <ion-item lines="none">
+        <ion-label class="ion-text-wrap">
+            <p>{{ getEnumDescription(currentJob.systemJobEnumId) }}</p>
```

Comment:
We won't use getter to fetch the enum description, we are getting description directly on job.